ambry-store/src/test/java/com.github.ambry.store/BlobStoreCompactorTest.java

```diff
@@ -592,6 +602,12 @@ public void interspersedDeletedAndExpiredBlobsTest() throws Exception {
     // there should be no temp files
     assertEquals("There are some temp log segments", 0,
         tempDir.listFiles(BlobStoreCompactor.TEMP_LOG_SEGMENTS_FILTER).length);
+    long expectedSavedBytes =
```

Comment:
some commonality in what is being asserted between tests - did you want to combine these into a method?

Reply:
Good idea - done